src/main/webapp/WEB-INF/tags/answerWith.tag

```diff
@@ -7,7 +7,7 @@
 <%@attribute name="commentVotes" type="org.mamute.model.CommentsAndVotes" required="true" %>
 <section class="post-area ${answer.isVisibleForModeratorAndNotAuthor(currentUser.current) ? 'highlight-post' : '' }">
 	<div class="post-meta">
-		<tags:voteFor item="${answer}" type="${t['answer.type_name']}" vote="${vote}"/>
+		<tags:voteFor item="${answer}" type="${answer.typeName}" vote="${vote}"/>
```

Comment:
Can you check if any other places are using `${t['*.type_name']}`? If no places are using we can remove those from our `mamute-messages*.properties` files